app/templates/reports/reports.html

```diff
@@ -70,6 +70,7 @@ <h2 class="ui header">
                             <th>Vehicle ID</th>
                             <th>License Plate</th>
                             <th>Duration</th>
+                            <th>Weather conditions during incident</th>
```

Comment:
Lol Nancy had the exact opposite comment

@nancywong @kongsally what should I do?